README.md

```diff
@@ -1 +1,25 @@
-EXPERIMENTAL WIP Json library based on a port of the [Jawn Parser](https://github.com/non/jawn) built on Arrow.
+# Helios
+
+Json library based on a port of the [Jawn Parser](https://github.com/non/jawn) built on Arrow.
```

Comment:
We need a better introduction here that appeals to users better than this. //cc @maureenelsberry

Reply:
I suggest writing a brief bit, starting with:

Helios is a Json library used to transform Json text into a model and vice versa. It's based on a part of the [Jawn Parser](https://github.com/non/jawn) built on Arrow, a Functional companion to Kotlin's Standard Library. 

Then talk a little bit about *why* people would use this? Use a sentence or two to solve the issue that the library is created for - ease of use, time, whatever it may be. 

I can edit the content when it's created - but can't really create much more than that as I'm not really deeply familiar with Helios at this point @AdrianRaFo @raulraja